src/decisionengine_modules/NERSC/sources/NerscSFApi.py

```diff
@@ -62,43 +61,51 @@ def check_accesstoken(self, nersc_user):
         except KeyError:
             self.logger.error(f"Unknown user '{nersc_user}', exiting")
             return None
-
+        print(nersc_user)
```

Comment:
Just a check - is this print statement supposed to be there or was it an artifact of debugging?